frontend/src/api/axios.test.js

```diff
@@ -0,0 +1,88 @@
+let expect = require('expect');
+let moment = require('moment');
+
+let users = [
+  {
+    username: 'karabas',
+    password: 'barabas',
+    name: 'Karabas Barabas',
+    email: 'root@nasa.gov',
+    age: 22,
+    userId: 'karabas1234567890',
+    dateCreated: moment(new Date())
+  }, {
+    username: 'barabas',
+    password: 'barabas',
+    name: 'Karabas Barabas',
+    email: 'root@nasa.com',
+    age: 22,
+    userId: 'karabas1234567890',
+    dateCreated: moment(new Date())
+  },
+];
+
+function checkUserAndPassword(accountData) {
+  let result = 'username_does_not_exist';
+  users.forEach(function (user) {
+    if (user.username === accountData.username) {
+      if (user.password === accountData.password) {
+        result = 'auth_ok';
+        return result;
+      } else {
+        result = 'wrong_password';
+        return result;
+      }
+    } 
+  });
+  return result;
+}
+
+function ifDataAlreadyExist(accountData) {
+  let result = 'ok';
+  users.forEach(function (user) {
+    if (user.username === accountData.username) {
+      result = 'username_already_exist';
+    }
+    if (user.email === accountData.email) {
+      result = 'email_already_exist';
+    }
+  });
+  return result;
+}
+
+// TEST checkUserAndPassword 1
+expect(
```

Comment:
This is a simple test file containing the same functions from axios.js I used for testing from node.js